engine/src/flutter/docs/contributing/Compiling-the-engine.md

```diff
@@ -117,11 +121,11 @@ For `--runtime-mode=profile` build, please also consider adding `--no-lto` optio
 
 These steps build the engine used by `flutter run` for iOS devices.
 
-Run the following steps, from the `src` directory created in the steps above:
+Run the following steps, from the `engine/src` directory:
 
-1. `git pull upstream main` in `src/flutter` to update the Flutter Engine repo.
+1. `git pull upstream master` to update the Flutter repo.
```

Comment:
I know this is old documentation, but why do we have different git instructions than from above?  Isn't it always going to be the same:

(assuming you are on origin/master)
* git fetch upstream
* git rebase upstream/main
* gclient sync -D
* et build

https://github.com/flutter/flutter/blob/5a23a2a3c4b08d3e13662191f1996fd316282ab9/docs/contributing/Tree-hygiene.md?plain=1#L140-L153